src/commands/game_commands/leaderboard.ts

```diff
@@ -27,8 +27,8 @@ enum LeaderboardAction {
 }
 
 const leaderboardQuotes = [
-    `Want your name to show up on the leaderboard? See \`${process.env.BOT_PREFIX}help leaderboard\``,
-    `Want to see the next page of the leaderboard? See \`${process.env.BOT_PREFIX}help leaderboard\``,
+    `Want your name to show up on the leaderboard? See ${process.env.BOT_PREFIX}help leaderboard`,
```

Comment:
Why get rid of code line here?

Reply:
They're footers so the code line doesn't work